astroObjectAnalyser/DataAnalysis/catalogues.py

```diff
@@ -128,4 +136,21 @@ def get_objects_image(self, image, cat, mask, cut_fixed=30):
             if (xc-numPix > 0) and (xc+numPix < nx) and (yc-numPix > 0) and (yc+numPix < ny):
                 cutout = image[int(xc-numPix):int(xc+numPix+1), int(yc-numPix):int(yc+numPix+1)]
                 list.append(cutout)
-        return list
\ No newline at end of file
+        return list
+    
+    def check_objects_saturation(self, cat_mask, quiet = False):
+        flags = cat_mask['FLAGS']
```

Comment:
can you make doc strings, i.e.
:param cat_mask: blabla
:param quiet: blabla
:return: blabla